main.tf

```
@@ -17,6 +17,9 @@ data "aws_caller_identity" "default" {
 data "aws_region" "default" {
 }
 
+data "aws_partition" "current" {
```

```suggestion
data "aws_partition" "default" {
```
yes, let's be constistent and use `default` name for resource